index.html

```diff
@@ -0,0 +1,434 @@
+                />Tasks
+              </li>
+              <li>
+                <img
+                  class="simple-icon"
+                  src="./icons/account-group.png"
+                  title="simple-icon"
+                  alt="communities icon"
+                />Communities
+              </li>
+            </ul>
+          </div>
+          <div class="setting-bar">
+            <ul class="setting-list nav-styles">
+              <li>
+                <img
+                  class="simple-icon"
+                  src="./icons/cog.png"
+                  title="simple-icon"
+                  alt="settings icon"
+                />Settings
+              </li>
+              <li>
+                <img
+                  class="simple-icon"
+                  src="./icons/help-box.png"
+                  title="simple-icon"
+                  alt="support icon"
+                />Support
+              </li>
+              <li>
+                <img
+                  class="simple-icon"
+                  src="./icons/shield-check.png"
+                  title="simple-icon"
+                  alt="privacy icon"
+                />Privacy
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="header">
+          <div class="top-header">
+            <div class="search-bar">
+              <div class="search-img clickable">
+                <input
+                  class="search-icon"
+                  src="./icons/magnify.svg"
+                  type="image"
+                  alt="image"
+                  form="myform"
+                  aria-label="my-form"
+                />
+              </div>
+              <form id="myform" method="get" action="#">
+                <input
+                  class="search-input"
+                  id="searchresult"
+                  name="searchresult"
+                  type="text"
+                  value="searchresult"
+                  aria-label="searchresult"
+                /><br /><br />
+              </form>
+            </div>
+            <div class="profile-bar">
+              <div class="clickable">
+                <img
+                  class="bell-icon"
+                  src="./icons/bell-ring-outline.svg"
+                  title="bell-icon"
+                  alt="notification icon"
+                />
+              </div>
+              <div class="clickable">
+                <img
+                  class="profile-pic-small"
+                  src="me.jpg"
+                  title="profile-pic-small"
+                  alt="profile picture"
+                />
+              </div>
+              <a
+                class="clickable3"
+                href="https://app.netlify.com/teams/abdulsomod1"
+                title="clickable2"
+              >
+                <div class="clickable">TENIFAYO ABDULSOMOD1</div>
+              </a>
+            </div>
+          </div>
+
+          <div class="bottom-header">
```

Comment:
take a look at this place, there can't be bottom header, you should try and use more descriptive name for that